src/pages/Create.js

```diff
@@ -0,0 +1,424 @@
+import React, { useState, useEffect, useMemo, useCallback, useRef } from 'react';
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+import IconButton from '@material-ui/core/IconButton';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import Popper from '@material-ui/core/Popper';
+import TextField from '@material-ui/core/TextField';
+import Grid from '@material-ui/core/Grid';
+import { makeStyles } from '@material-ui/core/styles';
+import ButtonGroup from '@material-ui/core/ButtonGroup';
+import FormatBoldIcon from '@material-ui/icons/FormatBold';
+import FormatItalicIcon from '@material-ui/icons/FormatItalic';
+import FormatUnderlinedIcon from '@material-ui/icons/FormatUnderlined';
+import TextFields from '@material-ui/icons/TextFields';
+import InsertLink from '@material-ui/icons/InsertLink';
+import Close from '@material-ui/icons/Close';
+import CreateMenu from "../components/CreateMenu";
+import { createEditor, Editor, Transforms, Text, Range } from "slate";
+import {Slate, Editable, withReact, useSlate, ReactEditor} from "slate-react";
+import { useTheme } from '@material-ui/core/styles';
```

Comment:
small nit, and maybe a formatter will do this automatically, but grouping imports is nice for quicker reference. 

I believe a formatted link prettier will do this via alphabetizing the imports.